tests/errors/vrf-invalid-lb.yml

```diff
@@ -0,0 +1,28 @@
+#
+# Invalid attributes used on VRF loopbacks
+#
+---
+defaults:
+  device: iosv
+
+module: [ bgp, vrf ]
+bgp.as: 65000
+
+vrfs:
+  red:
+    loopback: True
+  green:
+    loopback:
+      ipv4: 192.168.100.1/24
+      ospf.cost: 13
+  yellow:
+    loopback: 192.168.100.0/24
+
+nodes:
+  r1:
+    vrfs:
+      blue:
+        loopback:
+          bgp.advertise: False
```

Comment:
- Should add a node-level VRF loopback ipv4/v6 address
- Could include a subnet prefix error case